### PR TITLE
add status code, mimetype to response

### PR DESCRIPTION
**Pull request by B0pol**
It was always text/plain and HTTP 200 OK even when there was an error. I fixed that.

### CODE REVIEW

File: src/main/java/me/kavin/piped/Main.java [hunk 1]
```diff
@@ -38,7 +40,7 @@ public static void main(String[] args) throws Exception {
 		QueryStringDecoder query = new QueryStringDecoder(req.uri());
 
 		try {
-		    return writeResponse(res, query.parameters().get("hub.challenge").get(0), 200, "private", start);
+		    return writeResponse(res, query.parameters().get("hub.challenge").get(0), APPLICATION_XML, 200, "private", start);
```
Comment (FireMasterK) [inline src/main/java/me/kavin/piped/Main.java:43]: This should be `TEXT_PLAIN`

**Review by FireMasterK — APPROVED** [review]
LGTM! Good Job!

File: src/main/java/me/kavin/piped/Main.java (hunk 1 above)
Comment (FireMasterK) [reply src/main/java/me/kavin/piped/Main.java:43]: This is because the query parameter is just a random number/string.

Comment (B0pol) [reply src/main/java/me/kavin/piped/Main.java:43]:
```suggestion
		    return writeResponse(res, query.parameters().get("hub.challenge").get(0), TEXT_PLAIN, 200, "private", start);

```

Comment (B0pol) [reply src/main/java/me/kavin/piped/Main.java]: Please next time use diff, it's better to preview and we can commit if the change is small (like this one)

**Pull request merged**